packages/meditrak-app/ios/version-sync.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash -e
 
-# See https://medium.com/@andr3wjack/versioning-react-native-apps-407469707661
+# See http://web.archive.org/web/20170111215845/https://medium.com/@andr3wjack/versioning-react-native-apps-407469707661
```

Comment:
Shows how long ago we did this! I personally have never actually used the version sync, and always find it easier to just update it manually. Suggest deleting this file and any related mentions in the readme